repository_test.go

```diff
@@ -73,3 +73,45 @@ func TestRepositoryIsEmpty(t *testing.T) {
 		t.Error("Repository should not be empty")
 	}
 }
+
+func TestRepositoryChangePassword(t *testing.T) {
+	testPassword := "this_is_a_password"
+	newPassword := "this_is_another_password"
+
+	dir, err := ioutil.TempDir("", "knoxite")
+	if err != nil {
+		t.Errorf("Failed creating temporary dir for repository: %s", err)
```

Comment:
Instead of a `t.Errorf` followed by a return, I would suggest using `t.Fatalf` throughout this change.